packages/playground/src/ds/createDesignSystem.ts

```diff
@@ -154,7 +155,7 @@ export const createDesignSystem = (sandbox: Sandbox) => {
       createCodePre.setAttribute("tabindex", "0")
       const codeElement = document.createElement("code")
 
-      codeElement.innerHTML = code
+      codeElement.innerHTML = htmlEscape(code)
```

Comment:
Can this just be:

```suggestion
      codeElement.textContent = code
```

Then there's no need to write any escaping code.

Reply:
Good call

Reply:
Updated!